tests/test_repository_tool.py

```diff
@@ -113,6 +113,13 @@ def test_init(self):
                       'repository_directory', storage_backend, 3, 'targets_directory')
     self.assertRaises(securesystemslib.exceptions.FormatError, repo_tool.Repository,
                       'repository_directory', 'metadata_directory', 3, storage_backend)
+    self.assertRaises(securesystemslib.exceptions.FormatError, repo_tool.Repository,
+                  'repository_directory/', 'metadata_directory/', 'targets_directory/',
+                  storage_backend, repository_name, use_timestamp_length=3)
```

Comment:
Indentation here is off:
```suggestion
    self.assertRaises(securesystemslib.exceptions.FormatError, repo_tool.Repository,
                      'repository_directory/', 'metadata_directory/', 'targets_directory/',
                      storage_backend, repository_name, use_timestamp_length=3)
```